client/html/src/Client/Html/Basket/Standard/Standard.php

```diff
@@ -401,6 +401,12 @@ public function addData( \Aimeos\MW\View\Iface $view, array &$tags = [], string
 		}
 
 		if( empty( $params ) === false ) {
+			foreach( $params as $key => $val ) {
+				if( is_null( $val ) ) {
+					unset( $params[$key] );
+				}
+			}
+
 			$view->standardBackUrl = $view->url( $target, $controller, $action, $params, [], $config );
```

Comment:
using `array_filter( $params )` in this line has the same effect

Reply:
Yes and no, as `array_filter` removes empty (`''`) values, too. However, in this case I can't see a scenario where this could be useful, so I will change the PR.